scripts/dispersion.py

```diff
@@ -7,33 +7,28 @@
 """
 
 import argparse
-import os
-from euphonic.data.bands import BandsData
-from euphonic.data.phonon import PhononData
+import warnings
+
+from typing import List
+
 from euphonic.plot.dispersion import output_grace, plot_dispersion
 
+from .utils import load_data_from_file, get_args_and_set_up_and_down, matplotlib_save_or_show
```

Comment:
Can we move `.utils` to the main Euphonic package (e.g. `euphonic/script_utils.py`)? I can imagine users moving the scripts out of this directory and then wondering why they don't work

Reply:
Yes, I have moved it